sensor/LocalDomainSensor.js

```diff
@@ -72,6 +74,7 @@ class LocalDomainSensor extends Sensor {
     async globalOff() {
         try {
             await unlinkAsync(LOCAL_DOMAIN_FILE);
+            await unlinkAsync(ADDN_HOSTS_FILE);
```

Comment:
You can just delete the add host config file from FILTER_DIR, leaving the real hosts file intact.